wdn/templates_4.1/less/layouts/forms.less

```diff
@@ -1,4 +1,11 @@
 button,
+html [type="button"],
```

Comment:
I was restoring the removed styles directly from [normalize.css](https://github.com/necolas/normalize.css/blob/master/normalize.css#L278). However, the explicit scoping to `html` is to [prevent a WebKit audio/video control bug in Android 4](https://github.com/necolas/normalize.css/blob/master/normalize.css#L272-L273) caused by [-webkit-appearance: button](https://github.com/necolas/normalize.css/blob/master/normalize.css#L281). This fix is still part of normalize.css and still present in our framework, making the `html` unnecessary. I will remove.